domain/src/main/kotlin/no/nav/su/se/bakover/domain/oppdrag/Utbetalingskjøreplan.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.su.se.bakover.domain.oppdrag
+
+/**
+ * Fra dokumentasjon fra Oppdrag: Gjør det mulig å velge om delytelsen skal beregnes/utbetales i henhold til kjøreplanen eller om dette skal skje idag.
+ * Verdien 'N' medfører at beregningen kjøres idag. Beregningen vil bare gjelde beregningsperioder som allerede er forfalt.
+ *
+ * Tidligere var denne alltid satt til [NEI], men etter vi la til automatisk/manuell regulering ønsket vi å etterbetale sammen med neste kjøring.
+ */
+enum class Utbetalingskjøreplan {
```

Comment:
Kunne eventuelt brukt et ord som forteller noe om effekten for oss? Kanskje noe i duren av "etterbetal umiddelbart" vs "etterbetal sammen med neste utbetaling" eller lignende kunne fungert?

Reply:
Ja, i domenet er dette mer riktig. Enumen kunne kanskje hett Etterbetalingstidspunkt eller Etterbetalingsplan. Er vi forresten helt sikre på at dette gjelder kun for etterbetaling? Bør kanskje ta en ektra runde med oppdrag og forsikre oss om dette?